internal/host/statemachine.go

```diff
@@ -438,5 +438,17 @@ func NewHostStateMachine(th *transitionHandler) stateswitch.StateMachine {
 		})
 	}
 
+	// Noop transitions for day2 hosts
+	for _, state := range []stateswitch.State{
```

Comment:
You don't have to create a loop for only one transition.

Reply:
fixed